src/gil.rs

```diff
@@ -290,7 +293,7 @@ static POOL: ReferencePool = ReferencePool::new();
 
 /// A guard which can be used to temporarily release the GIL and restore on `Drop`.
 pub(crate) struct SuspendGIL {
-    count: usize,
+    count: isize,
```

Comment:
Now that `acquire_gil` is not a thing, this count is limited to recursive stack frames, so I wonder if `i16` or `i32` might be better?

Reply:
Since this is a single value (and not safe an array of them), I think anything below register size (hence `?size`) should not make a meaningful difference? With `i16` for example, I would want to audit the code for overflows and use checked operations.

I could settle for `i32`, but this already is `i32` for 32 bit platforms and I don't think it will make difference on 64 bit hardware.

Reply:
Fair point on register size, let's leave this as it is. 👍